src/Parser.js

```diff
@@ -392,52 +386,47 @@ var styleFuncs = [
  *   small text {\Large large text} small text again
  * It is also used for \left and \right to get the correct grouping.
  *
- * @return {?ParseResult}
+ * @return {?ParseNode}
  */
-Parser.prototype.parseImplicitGroup = function(pos) {
-    var start = this.parseSymbol(pos);
+Parser.prototype.parseImplicitGroup = function() {
+    var start = this.parseSymbol();
 
-    if (!start || !start.result) {
+    if (start === null) {
```

Comment:
OK, I changed this occurrence. Using `==` for `null` only is certainly better than always using it. For a moment I feared I might have to change many other strict comparisons as well.